applications/StructuralMechanicsApplication/custom_elements/shell_thin_element_3D4N.cpp

```diff
@@ -1853,19 +1853,18 @@ namespace Kratos
 			double l_13 = std::sqrt(inner_prod(r_13, r_13));
 
 			Vector e_24 = Vector(r_24 / l_24);
-			const Vector vec1 = Vector(MathUtils<double>::CrossProduct(r_13*-1.0, e_24)); // NOTE: This operation is marked as deprecated by the compiler, but it is not deprecated...not my fault
-                        const Vector vec2 = Vector(MathUtils<double>::CrossProduct(r_13, e_24));
-                        
-			const double d_24 = std::sqrt(inner_prod(vec1, vec2));
-			const double d_13 = std::sqrt(inner_prod(vec1, vec2));
+            const Vector vec1 = Vector(MathUtils<double>::CrossProduct(r_13, e_24));

```

Comment:
Vector vec1(3);
     MathUtils<double>::CrossProduct(vec1,r_13,e_24); //not changing order since you have -1